### PR TITLE
Jl calendar day bug

### PR DESCRIPTION
**Pull request by jleonardw9**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -216,7 +216,7 @@ def visits_select_options(arm, pages)
       (beginning_visit..ending_visit).each do |visit_number|
         visit_group = arm.visit_groups[visit_number - 1]
 
-        if visit_group.day.present?
+        unless visit_group.day.nil?
```
Comment (jayhardee9) [inline app/helpers/application_helper.rb]: `if visit_group.day` would be equivalent, and a little more understandable, in my opinion.

Comment (wtholt) [inline app/helpers/application_helper.rb]: I'm fine either way. I think I prefer it the way it is. Both ways make sense to me.

Comment (jleonardw9) [inline app/helpers/application_helper.rb]: I accidentally changed that when I was experimenting. I'll change it back.

Comment (jleonardw9) [inline app/helpers/application_helper.rb]: Reverted back to original.

File: app/models/arm.rb [hunk 2]
```diff
@@ -48,6 +48,14 @@ class Arm < ActiveRecord::Base
   validates :visit_count, numericality: { greater_than: 0 }
   validates :subject_count, numericality: { greater_than: 0 }
 
+  validate do |arm|
```
Comment (kyle-glick) [inline app/models/arm.rb]: Is there a better way of doing this? This seems too hard-coded

Comment (jleonardw9) [inline app/models/arm.rb]: I agree, but I couldn't figure out another way to make the validations chain back down to the visit groups. Let's think on it and try to refactor down the road.

**Pull request merged**